internal/docs/pubsub/design.md

```diff
@@ -254,52 +248,28 @@ func NewClient(ctx, projectName string) (*Client, error) {
 
 // OpenTopic opens an existing topic on the pubsub server and returns a Topic
 // that can be used to send messages to that topic.
-func (c *Client) OpenTopic(ctx context.Context, topicName string, opts *TopicOptions) (*pubsub.Topic, error) {
-	if opts == nil {
-		opts = defaultTopicOptions
-	}
+func (c *Client) OpenTopic(ctx context.Context, topicName string) (*pubsub.Topic, error) {
 	rt, err := c.rawclient.Topic(ctx, topicName)
 	if err != nil {
 		return err
 	}
-	t := &topic{
-		rawTopic: rt,
-		opts: opts,
-	}
-	return pubsub.NewTopic(t, opts.TopicOptions)
+	t := &topic{ rawTopic: rt }
+	return pubsub.NewTopic(t)
 }
 
 // OpenSubscription opens an existing subscription on the server and returns a
 // Subscription that can be used to receive messages.
-func (c *Client) OpenSubscription(ctx context.Context, subscriptionName string, opts *SubscriptionOptions) (*pubsub.Subscription, error) {
-	if opts == nil {
-		opts = defaultSubscriptionOptions
-	}
+func (c *Client) OpenSubscription(ctx context.Context, subscriptionName string) {
```

Comment:
I suspect that driver constructors will still want to have options for provider-specific extensibility.

Reply:
Okay. Adding the nils back in.

Reply:
You have added back the `nil`s, but not the parameter.

Reply:
Done.